app/views/contexts/section_summary_context.py

```diff
@@ -236,3 +307,52 @@ def _get_safe_page_title(self, title):
         return (
             safe_content(self._schema.get_single_string_value(title)) if title else ""
         )
+
+    @staticmethod
+    def _get_add_or_edit_blocks_primary(
+        primary_person_block: Mapping[str, Any]
+    ) -> tuple[str, str]:
+        primary_person_edit_block_id = primary_person_block["add_or_edit_block"]["id"]
+        edit_block_id = primary_person_block["add_or_edit_block"]["id"]
+
+        return primary_person_edit_block_id, edit_block_id
+
+    def _get_related_answers(self, current_list: ListModel) -> dict[str, list]:
```

Comment:
We need to handle related answers the same way a normal answer on a summary is handled. The only difference we have here is, answers have a list item id, everything else should behave the same way. You will need to extend Summary Group > Block > Answer classes to handle list item answers.

Reply:
Related answers now handled as normal answers.